### DISCUSSION TITLE
commit build files to version control?

### DISCUSSION THREAD

**Discussion by httnn** [tinacms/tinacms#3663] [Help] [answered]
hey! are there any downsides to committing the output of `tinacms build` to version control and re-running it manually only when there are changes to the schema? the command takes a long time on the CI and i'd like to avoid it for pure content changes if possible.

**Comment by jeffsee55** [comment, marked as answer]
You could certainly do that, let us know how it works out for you! I'd also just be careful to not that the config could change due to any imports you have in your config file.
